functions/taskrouterListeners/adjustCapacityListener.private.ts

```diff
@@ -45,41 +45,30 @@ export const shouldHandle = (event: EventFields) => eventTypes.includes(event.Ev
  * @param event
  */
 export const handleEvent = async (context: Context<EnvVars>, event: EventFields) => {
-  try {
-    const {
-      EventType: eventType,
-      WorkerSid: workerSid,
-      TaskChannelUniqueName: taskChannelUniqueName,
-    } = event;
-    if (taskChannelUniqueName !== 'chat') return;
-    console.log(`===== Executing AdjustCapacityListener for event: ${eventType} =====`);
-    const serviceConfig = await context.getTwilioClient().flexApi.configuration.get().fetch();
-    const {
-      feature_flags: {
-        enable_manual_pulling: enabledManualPulling,
-        enable_backend_manual_pulling: enableBackendManualPulling,
-      },
-    } = serviceConfig.attributes;
+  const { WorkerSid: workerSid, TaskChannelUniqueName: taskChannelUniqueName } = event;
+  if (taskChannelUniqueName !== 'chat') return;
+  const serviceConfig = await context.getTwilioClient().flexApi.configuration.get().fetch();
+  const {
+    feature_flags: {
+      enable_manual_pulling: enabledManualPulling,
+      enable_backend_manual_pulling: enableBackendManualPulling,
+    },
+  } = serviceConfig.attributes;
 
-    if (enabledManualPulling && enableBackendManualPulling) {
-      const { path } = Runtime.getFunctions().adjustChatCapacity;
+  if (enabledManualPulling && enableBackendManualPulling) {
+    const { path } = Runtime.getFunctions().adjustChatCapacity;
 
-      // eslint-disable-next-line global-require,import/no-dynamic-require,prefer-destructuring
-      const adjustChatCapacity: AdjustChatCapacityType = require(path).adjustChatCapacity;
-      const body = {
-        workerSid,
-        adjustment: 'setTo1',
-      } as const;
+    // eslint-disable-next-line global-require,import/no-dynamic-require,prefer-destructuring
+    const adjustChatCapacity: AdjustChatCapacityType = require(path).adjustChatCapacity;
+    const body = {
+      workerSid,
+      adjustment: 'setTo1',
```

Comment:
Why do we need to set capacity to 1 here? :thinking:

Reply:
We always set it back to 1 on certain events. With backend manual pulling, instead of trying to track capacity, we just always have it set to 1, them set it to the value required to take an extra task when manually pulling or accepting transfers, then set it back to 1 again when we're done.

There is no issue with being 'over capacity' other than not being able to accept more tasks, so this is simpler than trying to track capacity against tasks

Also this logic isn't changed in this PR